ruby/vampires.rb

```diff
@@ -0,0 +1,59 @@
+puts "How many employees will be processed?" 
+employees = gets.chomp.to_i #convert for arithmetic
+
+while employees > 0 
+	puts "What is your name?" # Question 1
+	name = gets.chomp
+	puts "How old are you?"   # Question 2
+	age = gets.chomp.to_i
+	puts "What year were you born?" # Question 3
+	yearmatch = true
+	byear = gets.chomp.to_i
+		if (2016-byear) == age # roughly check age by using current year
+			yearmatch # yearmatch is true still
+		else
+			yearmatch = false
+		end
+	puts "Our company cafeteria serves garlic bread. 
+	Should we order some for you?" # Question 4
+		order_g = gets.chomp
+		if order_g == "yes".upcase || order_g == "yes".capitalize! || order_g == "yes"
+			order_g = true #set to true
+		else 
+			order_g = false
+		end
+
+	puts "Would you like to enroll in the company's health insurance?" #Question 5
+		health = gets.chomp
+		if health == "yes".upcase || health == "yes".capitalize! || health == "yes"
+			health = true
+		else 
+			health = false
+		end
+	allergy = "" # Allergy variable, made accessible outside of loop for case statement
+	loop do
+		puts "List any allergies. Type 'done' when finished."
+		allergy = gets.chomp
+		if allergy == "sunshine" || allergy == "done"
+			break
+		end 
+	end
+
+	case # case statements
+	when (!yearmatch && (!order_g || !health))|| allergy == "sunshine"

```

Comment:
Match the following conditions in the order they're listed.' We first read this when starting off the challenge and we may try to hack it out with a basic if/elsif structure but we quickly learn that the first condition to pass will be the result even if a 'more true' version is below.  We may be tempted to switch the order for a working solution, but if we are to follow the constraints we need to be more creative. What if we made each individual scenario it's own respective if block? What would go inside each if block? Surely we can't just puts out the result otherwise our screen would be full of each vampire check that passed. What we could do is keep setting a variable result within each if block and then at the end of all the blocks display the result to look something like this
```
if condition_1
	result = 'low'
end

if condition_2
	result = 'medium'
end

if condition_3
	result = 'high'
end

puts result
```